### ISSUE TITLE
medic auto-call for injured teammates triggers when an enemy spy disguised as a teammate gets hurt

### ISSUE THREAD

**Issue by vlad55523** [mastercomfig/team-comtress-2#445]
### Description
 when an enemy spy disguised as a member of your team takes damage, medics can be pointed towards their location by the auto-call feature

### Checklist

- [x] None of the open *or* closed issues document this problem.
- [x] This is a bug, not a new idea.
- [x] This is objectively a bug, and not something that people commonly disagree on.
- [ ] This issue occurs on the latest Team Comtress release, or the `master` branch.
- [x] This issue occurs on the latest vanilla TF2 update.

- [x] This bug is in the code of the game, not in the textures, sounds, or other resources.
- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
- [ ] This issue occurs with DirectX level 100.
- [x] This issue occurs when using no custom content.
- [ ] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
- [x] This issue occurs when using a listen server (using the `map` command ingame).
- [x] This issue occurs when using a dedicated server.
- [ ] This bug is in the internal procedures or tooling (e.g. the build process).

### Reproduction

1. have a spy disguised as a member of your team
2. enable medic auto-call for injured people
3. you or any other member of your team has to deal damage to the disguised spy, you will see auto-call pop up

**Comment by Margen67** [comment]
The outline/glow on teammates also applies to disguised spies so this might be intentional.

**Comment by henriquecm** [comment]
Seems intentional, to mimic all behavior from a teammate

**Comment by agrastiOs** [comment]
Definitely looks intended.

**Comment by SirAngle** [comment]
Yeah, it doesn't appear like a bug to me. I think it's to make it less obvious they are a spy.
It seems intentional. 
Otherwise, if you see someone low HP without the med req you'd know they're a spy.

**Comment by 107zxz** [comment]
It would be intended if it triggered when the disguise was low on health, rather than the spy
